### PR TITLE
docs: add rdws secrets

### PR DESCRIPTION
**Pull request by yohanthshetty**
Updated docs/blog with rdws secrets.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/blogs/release-v126.en.md [hunk 1]
```diff
@@ -38,7 +38,38 @@ Copilot v1.26 brings several new features and improvements:
 
 ## Sidecar image build
 
-## Request-Driven Web Service secret support
+## Request-Driven Web Service secrets support
+You can now add your secrets(SSM Parameter Store or AWS Secrets Manager) to your App Runner service as environment variables using Copilot. First ensure that your secrets are tagged correctly. Then simply update your Request-Driven Web Service manifest with:
```
Comment (huanjani) [inline site/content/blogs/release-v126.en.md]:
```suggestion
You can now add your secrets (from SSM Parameter Store or AWS Secrets Manager) to your App Runner service as environment variables using Copilot. First ensure that your secrets are tagged correctly. Then simply update your Request-Driven Web Service manifest with:
```

File: site/content/blogs/release-v126.en.md [hunk 2]
```diff
@@ -38,7 +38,38 @@ Copilot v1.26 brings several new features and improvements:
...
+```yaml
+  secrets:
+    secret-name-as-env-variable: secret-name-from-aws
+```
+And deploy! Your service can now access the secret as an environment variable.
+For more detailed use of the secrets field:
```
Comment (huanjani) [inline site/content/blogs/release-v126.en.md]:
```suggestion
For more detailed use of the `secrets` field:
```

File: site/content/blogs/release-v126.en.md [hunk 3]
```diff
@@ -38,7 +38,38 @@ Copilot v1.26 brings several new features and improvements:
 
 ## Sidecar image build
 
-## Request-Driven Web Service secret support
+## Request-Driven Web Service secrets support
+You can now add your secrets (from SSM Parameter Store or AWS Secrets Manager) to your App Runner service as environment variables using Copilot. First ensure that your secrets are tagged correctly. Then simply update your Request-Driven Web Service manifest with:
```
Comment (Lou1415926) [inline site/content/blogs/release-v126.en.md]:
```suggestion
You can now add your secrets (from SSM Parameter Store or AWS Secrets Manager) to your App Runner service as environment variables using Copilot. 

Similar to other service types such as Load-Balanced Web Service, you need to first add the following tags to your secrets:
| Key                     | Value                                                       |
| ----------------------- | ----------------------------------------------------------- |
| `copilot-application`   | Application name from which you want to access the secret   |
| `copilot-environment`   | Environment name from which you want to access the secret   |

Then simply update your Request-Driven Web Service manifest with:
```

File: site/content/blogs/release-v126.en.md [hunk 4]
```diff
@@ -38,7 +38,38 @@ Copilot v1.26 brings several new features and improvements:
...
+```yaml
+  secrets:
+    secret-name-as-env-variable: secret-name-from-aws
+```
+And deploy! Your service can now access the secret as an environment variable.
+For more detailed use of the `secrets` field:
```
Comment (Lou1415926) [inline site/content/blogs/release-v126.en.md]: One "\n" will not render as a new line in md. We need two consecutive \n:
```suggestion
And deploy! Your service can now access the secret as an environment variable.

For more detailed use of the `secrets` field:
```

File: site/content/blogs/release-v126.en.md [hunk 5]
```diff
@@ -38,7 +38,38 @@ Copilot v1.26 brings several new features and improvements:
 
 ## Sidecar image build
 
-## Request-Driven Web Service secret support
+## Request-Driven Web Service secrets support
+You can now add your secrets (from SSM Parameter Store or AWS Secrets Manager) to your App Runner service as environment variables using Copilot. First ensure that your secrets are tagged correctly. Then simply update your Request-Driven Web Service manifest with:
+```yaml
+  secrets:
+    secret-name-as-env-variable: secret-name-from-aws
```
Comment (Lou1415926) [inline site/content/blogs/release-v126.en.md]:
```suggestion
    GITHUB_TOKEN: GH_TOKEN_SECRET
```

File: site/content/blogs/release-v126.en.md [hunk 6]
```diff
@@ -38,7 +38,38 @@ Copilot v1.26 brings several new features and improvements:
...
+```
+And deploy! Your service can now access the secret as an environment variable.
+For more detailed use of the `secrets` field:
+```yaml
+secrets:
+  # SecretsManager
```
Comment (Lou1415926) [inline site/content/blogs/release-v126.en.md]:
```suggestion
  # To inject a secret from SecretsManager.
```

File: site/content/blogs/release-v126.en.md [hunk 7]
```diff
@@ -38,7 +38,38 @@ Copilot v1.26 brings several new features and improvements:
...
+  DB_PASSWORD:
+    secretsmanager: 'demo/test/mysql:password::'
+  # You can substitute predefined environment variables to keep your manifest succinct.
+  DB_PASSWORD:
+        secretsmanager: '${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/mysql:password::'
+
+  # Option 2. Alternatively, you can refer to the secret by ARN.
+  DB: 'arn:aws:secretsmanager:us-west-2:111122223333:secret:demo/test/mysql-Yi6mvL'
+ 
+  #SSM Parameter Store
```
Comment (Lou1415926) [inline site/content/blogs/release-v126.en.md]:
```suggestion
  # To inject a secret from SSM Parameter Store
```

File: site/content/blogs/release-v126.en.md [hunk 8]
```diff
@@ -38,7 +38,38 @@ Copilot v1.26 brings several new features and improvements:
...
+  # (Recommended) Option 1. Referring to the secret by ARN.
```
Comment (Lou1415926) [inline site/content/blogs/release-v126.en.md]: Why is this recommended? I would probably have recommended option 2 😂

File: site/content/blogs/release-v126.en.md [hunk 9]
```diff
@@ -38,7 +38,38 @@ Copilot v1.26 brings several new features and improvements:
...
+  GITHUB_WEBHOOK_SECRET: 'arn:aws:ssm:us-east-1:615525334900:parameter/GH_WEBHOOK_SECRET'
+
+  # Option 2. Referring to the secret by name.
+  GITHUB_WEBHOOK_SECRET: GITHUB_WEBHOOK_SECRET
+```
+For more details see (../../docs/manifest/rd-web-service/#secrets-from-cfn).
```
Comment (Lou1415926) [inline site/content/blogs/release-v126.en.md]:
```suggestion
See the [manifest specification](../../docs/manifest/rd-web-service/#secrets). To learn more about injecting secrets into your services, see [the secrets page](../../docs/developing/secrets.en.md)
```

File: site/content/blogs/release-v126.en.md (hunk 3 above)
Comment (yohanthshetty) [reply site/content/blogs/release-v126.en.md]: Much Better! Thanks.

File: site/content/blogs/release-v126.en.md (hunk 8 above)
Comment (yohanthshetty) [reply site/content/blogs/release-v126.en.md]: :P At App Runner side we mostly use ARNs. Should I remove the `Recommended` word ?

File: site/content/blogs/release-v126.en.md (hunk 9 above)
Comment (yohanthshetty) [reply site/content/blogs/release-v126.en.md]: Great!

File: site/content/blogs/release-v126.en.md (hunk 8 above)
Comment (Lou1415926) [reply site/content/blogs/release-v126.en.md]: Yes let's remove "Recommended" 👍🏼 !

Comment (yohanthshetty) [reply site/content/blogs/release-v126.en.md]: Done!

**Review by iamhopaul123 — APPROVED** [review]
Looks good

**Review by Lou1415926 — APPROVED** [review]

File: site/content/blogs/release-v126.en.md [hunk 10]
```diff
@@ -70,7 +70,47 @@ With rollback alarms configured in your service manifest, each time you run `svc
...
+  # To inject a secret from SecretsManager.
+  # (Recommended) Option 1. Referring to the secret by name.
+  DB:
+    secretsmanager: 'demo/test/mysql'
+  # You can refer to a specific key in the JSON blob.
+  DB_PASSWORD:
+    secretsmanager: 'demo/test/mysql:password::'
+  # You can substitute predefined environment variables to keep your manifest succinct.
+  DB_PASSWORD:
+        secretsmanager: '${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/mysql:password::'
```
Comment (yohanthshetty) [inline site/content/blogs/release-v126.en.md:101]: Indentation

**Pull request merged**